x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart_with_tooltip.tsx

```diff
@@ -21,6 +21,44 @@ import { getControlsForDetector } from '../../get_controls_for_detector';
 import { MlAnnotationUpdatesContext } from '../../../contexts/ml/ml_annotation_updates_context';
 import type { SourceIndicesWithGeoFields } from '../../../explorer/explorer_utils';
 
+const percentFocusZoomPanelHeight = 0.051;
+const percentFocusChartHeight = 0.634;
+const percentContextChartHeight = 0.122;
```

Comment:
@alvarezmelissa87 @walterra what do you think about giving the context chart a fixed height, and have the focus chart fill the rest of the available height?

Reply:
Fixed height for the context chart is working well!